apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -414,6 +414,38 @@ fields("node") ->
                     desc => ?DESC(node_cookie)
                 }
             )},
+        {"process_limit",
+            sc(
+                range(1024, 134217727),
+                #{
+                    mapping => "vm_args.+P",
```

Comment:
Missing ‘readOnly’ so it will not be modified from UI.
Missing ‘default’
Same for other fields